go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/onosproject/config-models
 go 1.16
 
 require (
-	github.com/antchfx/xpath v1.2.0
+	github.com/SeanCondon/xpath v0.0.0-20220217125907-c2b75876708f
```

Comment:
is this to fix a bug in the upstream xpath?

Reply:
I needed to add a new function to the upstream XPath

Reply:
I doubt they'd take it because I have added a new function "set-contains" that's not in the XPath standard